hbase-server/src/main/java/org/apache/hadoop/hbase/util/FSTableDescriptors.java

```diff
@@ -158,25 +158,22 @@ public static TableDescriptorBuilder createMetaTableDescriptorBuilder(final Conf
         .setBlocksize(conf.getInt(HConstants.HBASE_META_BLOCK_SIZE,
                 HConstants.DEFAULT_HBASE_META_BLOCK_SIZE))
         .setScope(HConstants.REPLICATION_SCOPE_LOCAL)
-        // Disable blooms for meta.  Needs work.  Seems to mess w/ getClosestOrBefore.
-        .setBloomFilterType(BloomType.NONE)
```

Comment:
So we use ROW bloom which is the default. The above comment is no longer applies (or its equivalent API usage)

Reply:
Do you mean this comment?

         // Disable blooms for meta.  Needs work.  Seems to mess w/ getClosestOrBefore.